Lib/idlelib/editor.py

```diff
@@ -1008,12 +1007,16 @@ def open_recent_file(fn_closure=file_name):
     def saved_change_hook(self):
         short = self.short_title()
         long = self.long_title()
+        _py_version = ' (%s)' % platform.python_version()
         if short and long and not macosx.isCocoaTk():
             # Don't use both values on macOS because
             # that doesn't match platform conventions.
             title = short + " - " + long + _py_version
         elif short:
-            title = short
+            if short == "IDLE Shell":
+                title = short + platform.python_version()
```

Comment:
I left out the needed space in my suggestion.
```suggestion
                title = short + " " +  platform.python_version()
```